build.gradle.kts

```diff
@@ -8,10 +8,12 @@ import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 fun properties(key: String) = project.findProperty(key).toString()
```

Comment:
Please make default settings so that you don't need to configure it at first, you just launch the requests and tests are written in the tests directory inside the project